.dockerignore

```diff
@@ -1,8 +1,3 @@
 *
-!blueocean-commons/target/blueocean-commons.hpi
-!blueocean-dashboard/target/blueocean-dashboard.hpi
-!blueocean-plugin/target/blueocean.hpi
-!blueocean-rest/target/blueocean-rest.hpi
-!blueocean-rest-impl/target/blueocean-rest-impl.hpi
-!blueocean-web/target/blueocean-web.hpi
+!blueocean/target/plugins
```

Comment:
@michaelneale now that dependencies are assembled in the aggregator module, it's not needed to list them here